src/components/Header.tsx

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable @typescript-eslint/indent */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useContext, useState } from 'react';
+import { TodoContext } from '../contexts/TodoContext';
+// import { getNextId } from '../utils/getNextId';
+import { USER_ID } from '../constants';
+import { createTodo } from '../api/todos';
+import { Todo } from '../types/Todo';
+
+export const Header:React.FC = () => {
+  const [title, setTitle] = useState('');
+
+  const {
+    todos,
+    setTodos,
+    setErrorMessage,
+  } = useContext(TodoContext);
+
+  const uncompletedTodos = todos.filter(todo => !todo.completed).map(t => t.id);
+
+  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+  };
+
+  const handleSubmit = (
```

Comment:
I have possibility to create empty todo only with spaces
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/54936044/33f40b72-cb87-4c93-af5b-b17f02e11e79)